docs/walk-through.md

```diff
@@ -287,79 +342,38 @@ $ curl \
     --silent \
     --header "Content-Type: application/json; odata=minimalmetadata" \
     --data "{'duration':'PT5M'}" \
```

Comment:
Include a link to the REST API documentation?

Reply:
Where's the (public) REST API documentation?